raiden/network/proxies/token_network.py

```diff
@@ -959,20 +908,70 @@ def close(
         }
         log.debug('closeChannel called', **log_details)
 
-        checking_block = self.client.get_checking_block()
+        if signature != EMPTY_SIGNATURE:
```

Comment:
The proxy should just accept a data structure with the balance proof or None instead of requiring the user to provide all the argument. I will leave this for another PR.